src/Sentry/Internal/Hub.cs

```diff
@@ -99,6 +101,39 @@ public void WithScope(Action<Scope> scopeCallback)
 
         public void BindClient(ISentryClient client) => ScopeManager.BindClient(client);
 
+        public Transaction CreateTransaction(string name, string operation)
+        {
+            var trans = new Transaction(this, _options)
+            {
+                Name = name,
+                Operation = operation
+            };
+
+            var nameAndVersion = MainSentryEventProcessor.NameAndVersion;
+            var protocolPackageName = MainSentryEventProcessor.ProtocolPackageName;
+
+            if (trans.Sdk.Version == null && trans.Sdk.Name == null)
+            {
+                trans.Sdk.Name = Constants.SdkName;
+                trans.Sdk.Version = nameAndVersion.Version;
+            }
+
+            if (nameAndVersion.Version != null)
+            {
+                trans.Sdk.AddPackage(protocolPackageName, nameAndVersion.Version);
+            }
+
+            ConfigureScope(scope => scope.Transaction = trans);
```

Comment:
I believe by default we won't do this. The snippets I saw from JS require the caller to decide whether the transaction should be bound to the scope or not. 

For now this looks good and lets try like this.